### DISCUSSION TITLE
ESP32-4827S043 board issue with 1.3.5

### DISCUSSION THREAD

**Discussion by mignon-ne** [moononournation/Arduino_GFX#306] [General]
Hello,
Thank you for great work, I am very happy with the library.

However, I found the latest library has problem display incorrectly but if refresh it will solved.
I have tested on 1.3.4 and work fine, so far tested 1.2.8, 1.3.2 and 1.3.4 OK.
Only 1.3.3 and 1.3.5.

Please see attached clip.

https://github.com/moononournation/Arduino_GFX/assets/16378483/1b016538-5e15-4c68-9a30-f6df97eaafce

**Comment by moononournation** [comment]
Just released 1.3.6, please help to check is it ok now.

**Reply by mignon-ne** [reply]
https://github.com/moononournation/Arduino_GFX/assets/16378483/9f5423a6-d089-4085-81cc-75d31946ce0a

Thank you, it solved.
(Work with lvgl 8.3.7)
